packages/components/psammead-topic-tags/src/index.jsx

```diff
@@ -54,8 +55,13 @@ const SingleTopicTagItem = styled.div`
   }
 
   a > span {
+    display: inline-flex;
+    align-items: center;
```

Comment:
I'm not sure this is the intended usage of `inline-flex`, I've read here it is intended for laying out child elements as flex items but the `span` has no children: https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox#specifying_what_elements_to_lay_out_as_flexible_boxes ? 

Is there no other way of applying a `min-height` to the span, be interested hear what you tried?

In a similar vein, there are no child elements to align are there?

Reply:
Thanks @andrewscfc,

The min-height property could be applied here without `inline-flex`, but I used flexbox here so that the text of the topic tag could be centred vertically and horizontally using `align-items` and `justify-content` respectively; although there are no child HTML elements there is the text that the span contains. And instead of using `display: flex;` here I used `inline-flex` here because span is an inline element.